UnitedIncome/components
# [APP-1168] CompassCircleIllustration Component

### Description
> Please provide a brief description of your changes below.

Implements a circular compass SVG component with the option for animation. You can control the animation using the `isAnimated` and `animationDuration` props.

![rotattitititititionnn](https://user-images.githubusercontent.com/10888441/61550636-fce43380-aa20-11e9-978f-e8024a5bdb5e.gif)

You can even make it go super fast, if you wanted to.

![speeeed](https://user-images.githubusercontent.com/10888441/61550760-5b111680-aa21-11e9-941d-fdcbeafc43f7.gif)

### Testing Instructions
> What should be done in order to test this pull request, or re-create its effects.

* Load up the component in the browser and toggle the knobs on to start the animation.
* It should rotate at the desired speeds.

### Comments
> Any additional comments that will aid with the review process.

- [JIRA Ticket](https://unitedincome.atlassian.net/browse/app-*)
